packages/aragon-api/src/index.js

```diff
@@ -538,6 +556,76 @@ export class AppProxy {
         pluck('result')
       )
   }
+
+  /**
+   * send a forwarded action's information to the wrapper
+   *
+   * @param {string} actionId ID assigned to forwarded action in Forwarder
+   * @param {string} evmScript The execution script caught by the Forwarder
+   * @return {void}
+   */
+  newForwardedAction (actionId, evmScript) {
+    return this.rpc.send(
+      'update_forwarded_action',
+      [actionId, evmScript]
+    )
+  }
+
+  /**
+   * update a forwarded action's state the wrapper
+   *
+   * If the execution is pending, state = 0
+   * If the execution completed, state = 1
+   * If the execution failed, state = 2
+   *
+   * @param {string} actionId ID assigned to forwarded action in Forwarder
+   * @param {integer} state The current state of the forwarded action within the forwarder
+   * @param {string} [evmScript=''] The optionally updated execution script
```

Comment:
The EVMScript being updated here would be a case like DotVoting manipulating the execution script based on the voting results?

Also, maybe a better default is `null` (`undefined` also doesn't go through the RPC wire).

Reply:
We've decided that it's best to just pass this in every time, since we use the decoded "target" app as a key.